charts/jaeger-v2/templates/config-map.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: my-config
+  namespace: {{ .Release.Namespace }}
+  annotations:
+    "helm.sh/hook": pre-install
+data:
+  config.yaml: |
```

Comment:
isn't there a way in Helm to have config as a proper YAML file? When you use `|` the rest is treated as just a multi-line string, so you won't get any syntax highlighting or checking in the IDE.